src/python/pants/bsp/spec/base.py

```diff
@@ -46,7 +46,7 @@ def address_input(self) -> AddressInput:
                 f"Unknown URI scheme for BSP BuildTargetIdentifier. Expected scheme `pants`, but URI was: {self.uri}"
             )
         raw_addr = self.uri[len("pants:") :]
-        return AddressInput.parse(raw_addr)
+        return AddressInput.parse(raw_addr, description_of_origin="TODO(#14468)")
```

Comment:
Help appreciated @stuhood @tdyas (possibly in a followup). Tom suggested over DM that we refer to the config file the address comes from, although that information is not stored on this type.